src/class-elements/delete-error/cls-decl-field-delete-covered.template

```diff
@@ -21,8 +21,6 @@ info: |
     CoverParenthesizedExpressionAndArrowParameterList ultimately derives a
     phrase that, if used in place of UnaryExpression, would produce a
     Syntax Error according to these rules. This rule is recursively applied.
-
-flags: [onlyStrict]
```

Comment:
Should these tests include something in the `info` that explains why they _must not_  have an `onlyStrict` flag?

Reply:
ok, I'll do it